src/components/change-view.jsx

```diff
@@ -145,9 +152,17 @@ export default class ChangeView extends React.Component {
     return (
       <div className="change-view">
         {userCanAnnotate ? this.renderSubmission() : null}
-        <VersionistaInfo versions={this.props.page.versions} from={this.props.from} to={this.props.to}/>
+        <div className="utilities">
+          <VersionistaInfo versions={this.props.page.versions} from={this.props.from} to={this.props.to}/>
+          <DiffSettings
+            diffType={this.state.diffType}
+            onRemoveFormattingChange={this.handleRemoveFormattingChange}
+            removeFormatting={this.state.removeFormatting}
```

Comment:
I added this prop so that the checkbox could be a "controlled component", but not sure if it's necessary.

Reply:
So when I said:

> That way it’s easier to handle other sorts of settings we might wind up adding in the future.
> Then this would be an object representing a collection of settings, rather than a single one.

I was kinda thinking:

```js
<DiffSettings
  diffType={this.state.diffType}
  settings={this.state.diffSettings}
  onChange={this.handleDiffSettingsChange}
/>
```

…still controlled, but with a more abstract interface so we can add things.

Reply:
But anyway, yes, controlled makes sense. It’s most fitting with React.

Reply:
And `state.diffSettings` would be like `{removeFormatting: true}` or something like that.

Reply:
*facepalm* lol

Reply:
I'll re-think this, because I totally misunderstood

Reply:
Well you’re still at 80% of what I was thinking. Just pass around a different object; not sure anything else really needs to be structurally different.